src/main/java/com/clanki/commands/ReviewCommand.java

```diff
@@ -0,0 +1,49 @@
+package com.clanki.commands;
+
+import com.clanki.objects.Flashcard;
+import com.clanki.objects.FlashcardList;
+import com.clanki.ui.Ui;
+
+import java.util.ArrayList;
+import java.util.Objects;
+
+public class ReviewCommand extends Command {
+    private static void reviewCurrentFlashcard(Ui display, Flashcard flashcard) {
+        boolean isCurrentCardPassed = false;
+        display.print("Let's see the next flashcard!");
+        while (!isCurrentCardPassed) {
+            display.print("Question for the current flashcard is: ", flashcard.getQuestion());
```

Comment:
The current test error is because there is a space at the end here, while there isn't in the `EXPECTED.txt` file.